x-pack/plugins/security_solution/server/endpoint/ingest_integration.ts

```diff
@@ -57,9 +70,18 @@ export const getPackageConfigCreateCallback = (
     try {
       return updatedPackageConfig;
     } finally {
```

Comment:
nit -> We might think about sending a promise instead of counting on finally. I think that will be more readable too